.gitignore

```diff
@@ -73,3 +73,8 @@ __pycache__
 
 # prerequisites for updating ChibiOS
 /util/fmpp*
+
+sigul-planck
+# custom code added to hide my secret files
+secrets.c
+secrets.h
```

Comment:
```suggestion
```

Reply:
why? I added this following @drashna guide for secret macros. or maybe I didn't understand it.

Reply:
ok found it